.github/CONTRIBUTING.md

```diff
@@ -117,6 +117,23 @@ To link your Codecademy user profile to GitHub:
 - Typos/bugs
 - Plagiarism
 
+## How to keep a PR branch upto date
```

Comment:
Make the following change on line # 102:

~"Any tips for a Pull Request?"~ -> "Any tips for a Pull Request (PR)?"

Then, on this line, I suggest the following edit:

```suggestion
## How do we update a PR branch?
```